app/screens/home/search/team_picker.tsx

```diff
@@ -98,23 +96,18 @@ const TeamPickerIcon = ({size = 24, divider = false, setTeamId, teams, teamId}:
                     onPress={handleTeamChange}
                     type='opacity'
                     testID='team_picker.button'
+                    style={styles.teamContainer}
                 >
-                    <View style={[styles.teamContainer, divider && styles.border]}>
-                        <View style={[styles.teamIcon, {width: size, height: size}]}>
-                            <TeamIcon
-                                displayName={selectedTeam.displayName}
-                                id={selectedTeam.id}
-                                lastIconUpdate={selectedTeam.lastTeamIconUpdatedAt}
-                                textColor={theme.centerChannelColor}
-                                backgroundColor={changeOpacity(theme.centerChannelColor, 0.16)}
-                                selected={false}
-                                testID={`team_picker.${selectedTeam.id}.team_icon`}
-                                smallText={true}
-                            />
-                        </View>
+                    <View style={{flex: 1}}>
+                        <Text
+                            id={selectedTeam.id}
```

Comment:
shouldn't we add an ellipsis here? also why are we removing the icon?

Reply:
The ellipsis is handled:

![Screenshot_1734631074](https://github.com/user-attachments/assets/42ac73c5-e1ac-41d7-ae54-5a68481a1bd3)


And the icon removal is part of the new design